package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "@tbd54566975/dwn-sdk-js",
-  "version": "0.0.17",
+  "version": "0.0.18",
   "description": "A reference implementation of https://identity.foundation/decentralized-web-node/spec/",
   "type": "module",
   "types": "./dist/esm/src/index.d.ts",
   "engines": {
     "node": ">= 16"
   },
+  "main": "./dist/cjs/src/index.js",
```

Comment:
just wanted to comment that i'm not sure we necessarily need a `cjs` bundle. the esm ones should work in theory, as RN uses esm. you could try pointing the `main` field to the esm, or if that's a non starte you could try adding the [react-native directive](https://github.com/facebook/metro/issues/670#issuecomment-1099926849) instead of `main`

Reply:
nice find @shamilovtim! i'll def do that instead. @thehenrytsai am going to yank out the cjs before merging this

Reply:
actually going to leave the cjs as electron doesn't yet support esm. @shamilovtim any preference on whether you want the react native directive to point to esm or cjs? i imagine ejs but just wanted to be sure

Reply:
probably esm. no idea about ejs file types specifically